flow/options.flow.js

```diff
@@ -9,5 +9,6 @@ declare type Options = { // eslint-disable-line no-undef
     stubs?: Object,
     context?: Object,
     attrs?: Object,
-    listeners?: Object
+    listeners?: Object,
+    logModifiedComponents?: Boolean
```

Comment:
I think it is necessary to add  the types to the below files.

https://github.com/vuejs/vue-test-utils/blob/14c40e6af2517b5a058f84d8cea51a711d929924/packages/test-utils/types/index.d.ts#L142

https://github.com/vuejs/vue-test-utils/blob/14c40e6af2517b5a058f84d8cea51a711d929924/packages/server-test-utils/types/index.d.ts#L52